schema/deploy/shipping/views.sql

```diff
@@ -768,6 +768,27 @@ create or replace view shipping.hcov19_observation_v1 as
     ;
 
 
+/* The shipping.hcov19_observation_v1 view needs hCoV-19 visibility, so
+ * remains owned by postgres, but it should only be accessible by
+ * reportable-condition-notifier.  Revoke existing grants to every other role.
```

Comment:
I copied this comment from the `return_results` view. I don't think it's correct as is. Should `reportable-condition-notifier` have access to this view? By revoking access to `reporter`, don't we leave `fhir-processor` and `presence-absence-processor` with access, too? I'd like to discuss this with you tomorrow if possible.

Reply:
See db7f9df, which I just pushed to correct the comment and add a grant for hcov19-visibility.